keyboards/planck/keymaps/jetpacktuxedo/config.h

```diff
@@ -0,0 +1,43 @@
+#ifndef CONFIG_USER_H
+#define CONFIG_USER_H
+
+#include "config_common.h"
```

Comment:
This line is not actually needed, and should be removed. 

Specifically, this is already included by the keyboard's config.h file, which is included by the build script.